build.gradle

```diff
@@ -24,3 +24,9 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+tasks {
+    wrapper {
+        distributionType = Wrapper.DistributionType.ALL
+    }
+}
```

Comment:
Why this block?

Reply:
When you run `./gradlew wrapper --gradle-version=6.3` on the command-line without this block, the task will revert you back to the `-bin` version by default. I saw this project uses `-all` so this will avoid one extra manual change on future updates of the Gradle wrapper.

Reply:
OK, I think it is better to use `-bin` for scrcpy, so I'm ok to switch to `-bin`. What do you think?

Reply:
I believe `-bin` works perfectly fine as well. I can rebase my branch right now and drop the wrapper task configuration.

Reply:
Please check [`pr1263.rom1v`](https://github.com/Genymobile/scrcpy/commits/pr1263.rom1v) :wink: If you're ok with it, I merge.